src/app/components/forms/multi-check-box/terra-multi-check-box.component.html

```diff
@@ -1,18 +1,24 @@
 <div class="multiselect-container">
     <div class="multiselect-element" [ngClass]="{'disabled': inputIsDisabled}">
         <div class="multiselect">
-            <div class="multiselect-header">
+            <div class="multiselect-header"
```

Comment:
maybe rename all the css-selectors to match the component naming? multi-checkbox-

Reply:
That can be done in another pr

Reply:
ok thought it was new...